IgOmeProfiling_pipeline.py

```diff
@@ -300,6 +311,27 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
     parser.add_argument('--no_rpm_factor', action='store_true', help='Disable multiplication hits by factor rpm for normalization')
     parser.add_argument('--no_use_rpm_faa_scanning', action='store_true', help='Disable performance of scanning script with unique rpm faa file')
     parser.add_argument('--no_output_sequences_scanning', action='store_true', help='Disable storing the output sequences that had hits')
+    parser.add_argument('--use_positive_motifs_script', action='store_true', help='Before classification model, remain only positive motifs from the csv file')
+    parser.add_argument('--invalid_mix',type=str, default=None, help='A argument to know if there is compare to naive')

```

Comment:
Shouldn't the help be:
Sample name considered negative. e.g. "native"
?

Should default be "naive"?

Reply:
Not all the time there are naive samples.